src/sentry/api/endpoints/organization_teams.py

```diff
@@ -98,6 +99,10 @@ def get(self, request, organization):
             organization=organization, status=TeamStatus.VISIBLE
         ).order_by("slug")
 
+        if request.GET.get("is_not_member", "0") == "1":
+            user_teams = Team.objects.get_for_user(organization=organization, user=request.user)
+            queryset = queryset.exclude(id__in=[ut.id for ut in user_teams])
```

Comment:
You should be able to use a subquery expression here to avoid evaluating the `user_teams` query.

```python
from django.db.models import Subquery
queryset = queryset.exclude(id__in=Subquery(user_teams.values('id')))
```

Reply:
@markstory looks like `get_for_user` returns a list of teams: https://github.com/getsentry/sentry/blob/126d272d1e43a11f503e12ff744859d2da8a56dd/src/sentry/models/team.py#L64